mockingcase.test.js

```diff
@@ -131,5 +131,24 @@ describe('mockingcase', () => {
     });
   });
 
+  describe('default options when importing/requering package', () => {
+    test('default options', () => {
+      const mOcKiNgCaSe = require('./index').config({ onlyLetters: true });
+      const input = 'hello';
```

Comment:
```suggestion
      const input = 'hello123';
```
so the default options will remove numbers and expected output will be `hElLo`